roles/python-app/vars/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-python_app_python_exec: "{{ python_app_python3 | bool | ternary('python3', 'python') }}"
-python_app_pip_exec: "{{ pip_python3 | bool | ternary('pip3', 'pip') }}"
+python_app_python_exec: "{{ python_app_python3 | bool | ternary('python3.5', 'python') }}"
+python_app_pip_exec: "{{ pip_python3 | bool | ternary('pip3.5', 'pip') }}"
```

Comment:
feels like this would cause us upgrade problems.

Reply:
It will only cause problems when you go to 3.6. A bridge to cross later?